### PR TITLE
Checkbox thumbnail

### PR DESCRIPTION
**Pull request by lslachciak**

### CODE REVIEW

**Comment by lslachciak** [comment]
Current status:

I have problem with positioning check On

File: examples/mobile/UIComponents/components/controllers/checkbox-thumbnail-list.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+			</h1>
+		</div>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-divider ui-li-has-checkbox ui-li-has-checkbox-icon"> 
+					<input checked="checked" class="checkboxes" name="checkbox-1" type="checkbox" />
+					<div class="ui-li-checkbox-icon">
+						<img src="Thumbnail/thumbnail_001.jpg"/>
+					</div>
+					<div class="ui-li-text">
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/controllers/checkbox-thumbnail-list.html:33]: style="align-self:center" for ui-li-text

**Comment by lslachciak** [comment]
Ready for review:
image

File: examples/mobile/UIComponents/components/controllers/checkbox-thumbnail-list.html (hunk 1 above)
Comment (lslachciak) [reply examples/mobile/UIComponents/components/controllers/checkbox-thumbnail-list.html:33]: not needed anymore

**Review by korneliakobiela — APPROVED** [review]
Looks good

**Review by singa2000 — APPROVED** [review]
LGTM

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**